command/agent/consul/client.go

```diff
@@ -36,6 +36,9 @@ const (
 	// defaultMaxRetryInterval is the default max retry interval.
 	defaultMaxRetryInterval = 30 * time.Second
 
+	// defaultPeriodicInterval is the default periodic sync interval.
```

Comment:
Can you add a bit more to this comment. "The periodic sync reconciles state between the desired services and checks and the actual registered checks. This is done at an interval, rather than being purely edge triggered, to handle the case that the Consul agent's state may change underneath us".